### ISSUE TITLE
LARS learning rate when norm is less than threshold

### ISSUE THREAD

**Issue by shmurai** [chainer/chainer#8244]
When LARS is used, it is expected that LR given to the optimizer is already multiplied by eta.

https://github.com/chainer/chainer/blob/30fcfa67e6e21ff6682bcbda0fdd1543a203cae6/chainer/optimizer_hooks/gradient_lars.py#L101
Therefore, in the code above, if the norm of weight (p_norm) is less than the threshold, the actual LR applied to the parameter is lr * eta, where lr is the one user meant to use.
Is this the expected behavior?

**Comment by beam2d** [comment]
Thanks for catching the point! It looks `lr` should be used instead of `lr * eta` when `p_norm` is under the threshold. I'm not sure if the behavior is originally expected, but the implementation has not been changed since the first PR (#4237).

It's good to make `eta` configurable in GradientLARS. I'm wondering if we should also change the convention of treating the learning rate of SGD as `lr * eta`.

**Comment by shmurai** [comment]
I agree that we should change the convention. Current convention is somewhat confusing and it makes the situation more chaotic if LARS is to be applied to only a part of parameters.

FYI, LARS implementations of other frameworks:
- TensorFlow: https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/contrib/opt/python/training/lars_optimizer.py#L29-L176
- Keras: https://github.com/keras-team/keras-contrib/blob/master/keras_contrib/optimizers/lars.py#L5-L99
Both look to be using `lr` if `p_norm` is under the threshold.
